services/storage-users/pkg/config/config.go

```diff
@@ -99,9 +99,16 @@ type Drivers struct {
 	Local LocalDriver `yaml:",omitempty"` // not supported by the oCIS product, therefore not part of docs
 }
 
+// AsyncPropagatorOptions configures the async propagator
+type AsyncPropagatorOptions struct {
+	PropagationDelay time.Duration `yaml:"propagation_delay" env:"STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY" desc:"The delay between a change to tree and the start of the according treesize and treetime propagation."`
+}
```

Comment:
@mmattel do we need to make this clearer?

Reply:
I will check today, but I need to finish some private stuff.

Reply:
Done